pkg/block/fetcher.go

```diff
@@ -283,9 +291,22 @@ func (f *ConcurrentLister) GetActiveAndPartialBlockIDs(ctx context.Context, ch c
 	}
 	close(metaChan)
 
-	if err := eg.Wait(); err != nil {
-		return nil, err
+	multiError = multierror.Append(eg.Wait(), multiError)
```

Comment:
Shouldn't this be the other way around?

Reply:
Good catch! Swapped them around. Thanks!